package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
`imports[1]` in this case is `redbox-react`.  It gets passed to `react-transform-catch-errors` to report errors to the developer.  I'm not so much using `jspm` as I am `systemjs`, and I'm using it as a way to dynamically load arbitrary modules within the browser, without the need for a server + webpack.